osu-cass/cass-pelican
# Remove sidebar

This removes the sidebar completely. Here is what is looks like:

![Without sidebar](https://cloud.githubusercontent.com/assets/1390653/17413185/0789d5ea-5a35-11e6-80bb-98d42dfe04b2.png)

## Testing
1. Run `make html`
2. Run `make devserver`
3. Visit localhost:8000 and see that the sidebar was removed

@Kennric @kelnera @alxngyn @leian7 @athai
